### DISCUSSION TITLE
Possible Window Closable Issue

### DISCUSSION THREAD

**Discussion by theoreaves** [NativePHP/laravel#317] [Q&A]
I am opening a window like this:

        Window::open('GitPullForm')
            ->width(1024)
            ->height(768)
            ->url(route('git.pull'))
            ->rememberState()
            ->closable(false)
            ->minimizable(false)
            ->alwaysOnTop()
            ->title('Git Pull');

Which works, and the closable works great.  Too Great. LOL!  I can't close the window with
Window::close('Git Pull');  it stays open. But if I take out closable, it works. 

Is there a way to switch the state of closable from false to true after an event happens?

**Comment by simonhamp** [comment]
You can [now](https://github.com/NativePHP/laravel/pull/391) toggle whether a window is closable using the `Window::closable()` method.
